packages/twenty-front/src/modules/workflow/workflow-trigger/components/WorkflowEditTriggerCronForm.tsx

```diff
@@ -33,9 +33,9 @@ export const WorkflowEditTriggerCronForm = ({
   triggerOptions,
 }: WorkflowEditTriggerCronFormProps) => {
   const theme = useTheme();
-  const [errorMessages, setErrorMessages] = useState<{
-    [key: number]: string | undefined;
-  }>({});
+  const [errorMessages, setErrorMessages] = useState<
+    Record<string, string | undefined>
+  >({});
```

Comment:
To improve type-safety, we could define an interface for all the possible error messages:

```tsx
type FormErrorMessages = {
  CUSTOM: string | undefined;
  HOURS_hour: string | undefined;
  HOURS_minute: string | undefined;
  MINUTES: string | undefined;
  SECONDS: string | undefined;
}
```

```suggestion
  const [errorMessages, setErrorMessages] = useState<FormErrorMessages>({});
```